feat(embedding transactions): adding to traversal (#3840)
getsentry/sentry-docs: src/docs/product/sentry-basics/tracing/event-detail.mdx

</Alert>

> #### Traversing Transactions

Reviewer: @doralchan I don't think we should delete the traversal portion of the content. I feel that embedded transaction and traversal are two distinct (but similar) product features that have some value.

Reviewer 2: That was me. I have a potential solution that we can go over in our meeting this afternoon.

> #### Embedded Transactions

Reviewer: We may need the early adopter note here for embedded transaction feature.

</Alert>

> #### Embedding and Traversing Transactions

Reviewer 2: ```suggestion
#### Embedded Transactions
```

> In some cases, a span may be the parent of one or more transactions. You can identify which span has children with the expand callout. Clicking this icon will embed the first child transaction. When a transaction is embedded, the collapse callout will appear for you to hide it again.

Reviewer 2: I've also made some other edits here based on what the UI seems to be doing, but please confirm. "Callout" seems like a pretty UI/UX-specific term. Suggest sticking with icon. 

```suggestion
In some cases, a span may be the parent of one or more transactions. Spans that have children are identified by the expand icon. Clicking this icon will display the first embedded child transaction. When that transaction is displayed, the collapse icon will appear so that you can the transaction again.
```

> In some cases, a span may be the parent of one or more transactions. You can identify which span has children with the expand callout. Clicking this icon will embed the first child transaction. When a transaction is embedded, the collapse callout will appear for you to hide it again.

Reviewer 2: I've made some suggestions here based on what the UI seems to be doing, but please confirm. Also "callout" seems like a pretty UI/UX-specific term. Suggest sticking with icon. Also, looking at the UI, the tooltip text seems to be different now.

```suggestion
In some cases, a span may be the parent of one or more transactions. Spans that have children are identified by the expand icon, and clicking this icon will display the first embedded child transaction. When that transaction is displayed, the collapse icon will appear so that you can hide the transaction again.
```

> 
> You can also click any part of the row itself to investigate the span details more where some Span IDs will have a "View Child" or "View Children" button. These will potentially lead to another transaction or a list of transactions.

Reviewer 2: Are the buttons still a thing? 
```suggestion
You can also click any part of the row itself to investigate the span details further.
```

> #### Embedding and Traversing Transactions

Reviewer 2: ```suggestion
#### Fetching and Traversing Transactions
```

Reviewer: Instead of "Fetching", maybe "Inlining" would be a better word? 

For context, we're inlining (or embedding) transactions onto the span view (waterfall).

> In some cases, a span may be the parent of one or more transactions. Spans that have children are identified by the expand icon, and clicking this icon will display the first embedded child transaction. When that transaction is displayed, the collapse icon will appear so that you can hide the transaction again.

Reviewer 2: ```suggestion
In some cases, a span may be the parent of one or more transactions. Spans that have children are identified by the expand icon, and clicking this icon will fetch and display the first child transaction. When that transaction is displayed, the collapse icon will appear so that you can hide the transaction again.
```

> You can also click any part of the row itself to investigate the span details more where some Span IDs will have a "View Child" or "View Children" button. These will potentially lead to another transaction or a list of transactions.

Reviewer 2: ```suggestion
You can also click any part of the span row itself to investigate its details. In the details, some spans have a "View Child" or "View Children" button. Clicking these buttons will potentially lead to another transaction or list of transactions.
```
